src/main/java/mate/academy/bookstoreapp/model/CartItem.java

```diff
@@ -9,13 +9,17 @@
 import jakarta.persistence.ManyToOne;
 import jakarta.persistence.OneToOne;
 import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
+import lombok.ToString;
 
 @Entity
 @Table(name = "cart_items")
 @Getter
 @Setter
+@ToString
+@EqualsAndHashCode
 public class CartItem {

```

Comment:
add `@ManyToOne(fetch = FetchType.LAZY, optional = false)` for private ShoppingCart shoppingCart;